src/main/java/bio/terra/service/dataset/DatasetRequestValidator.java

```diff
@@ -372,6 +376,31 @@ public void validate(@NotNull Object target, Errors errors) {
             if (schema != null) {
                 validateSchema(schema, errors);
             }
+            validateRegion(datasetRequest, errors);
+        }
+    }
+
+    private void validateRegion(DatasetRequestModel datasetRequest, Errors errors) {
```

Comment:
I think this can be `static`
```suggestion
    private static void validateRegion(DatasetRequestModel datasetRequest, Errors errors) {
```